lib/tasks/youth.rake

```diff
@@ -0,0 +1,286 @@
+        housing_status: 'Housing Status',
+        other_agency_involvement: 'Is this youth involved with any other state Agencies?',
+        owns_cell_phone: 'Does this youth own a cell phone?',
+        secondary_education: 'Secondary Education',
+        attending_college: 'Is this youth attending college?',
+        health_insurance: 'Does this youth have health insurance?',
+        requesting_financial_assistance: 'Is this youth asking for or receiving financial assistance from us at the time of entering this data?',
+        client_dob: 'Youth Birthday',
+        staff_believes_youth_under_24: 'Staff believes youth is 24 or under?',
+        client_gender: 'Gender',
+        client_lgbtq: 'Is the youth LGBTQ+ ?',
+        client_race: 'Race (Check all that apply)',
+        client_ethnicity: 'Ethnicity',
+        client_primary_language: 'Primary Language',
+        pregnant_or_parenting: 'Is this youth Pregnant or Parenting?',
+        disabilities: 'Disabilities, check all that apply.',
+        how_hear: 'How did this Youth hear about us?',
+        financial_assistance_provided: 'Has this youth received any direct financial assistance?',
+        financial_assistance_type_provided: 'What kinds of Financial Assistance has this youth received, select all that apply.',
+        needs_shelter: 'Is this Youth in need of a shelter?',
+        referred_to_shelter: 'Did we refer them to a shelter?',
+        in_stable_housing: 'Is this youth currently in stable housing?',
+        stable_housing_zipcode: '(If stably housed) What is the zipcode of the stable housing this youth is in?',
+        referrals: 'Please check all applicable referrals',
+        youth_case_management: 'Do you have a case management meeting / intake meeting to log for today\'s date of engagement?',
+        PersonalID: 'Unique ID',
+      }
+    end
+
+    def race(race_string)
+      {
+        'AmIndAKNative' => 'American Indian or Alaska Native',
+        'Asian' => 'Asian',
+        'BlackAfAmerican' => 'Black or African American',
+        'NativeHIOtherPacific' => 'Native Hawaiian or Other Pacific Islander',
+        'White' => 'White / Caucasian',
+      }.invert[race_string]
+    end
+
+    def ethnicity(ethnicity_string)
+      {
+        0 => 'Non-Hispanic / Non-Latino',
+        1 => 'Hispanic / Latino',
+        8 => 'Unknown',
+        9 => 'Youth refused',
+        99 => 'Data not collected',
+      }.invert[ethnicity_string]
+    end
+
+    def gender(gender_string)
+      {
+        'Female' => 0,
+        'Male' => 1,
+        'Trans Male (MTF or Male to Female)' => 3,
+        'Trans Female (MTF or Male to Female)' => 2,
+        'Trans Male (FTM or Female to Male)' => 3,
+        'Trans Female (FTM or Female to Male)' => 2,
+        'Gender non-conforming (i.e. not exclusively male or female)' => 4,
+        'Unknown' => 8,
+        'Youth Refused' => 9,
+        'Data not collected' => 99,
+      }[gender_string]
```

Comment:
Curious to why you did `race` and `ethnicity` with `invert` and this one directly?

Reply:
There are some duplicates values in gender.